CHANGES.md

```diff
@@ -4,7 +4,8 @@ CKEditor 4 Changelog
 ## CKEditor 4.15.2
 
 Fixed Issues:
-
```

Comment:
This empty line should stay here for nicer formatting.

Reply:
Added empty line.